src/test/kotlin/cacophony/codegen/instructions/matching/InstructionMatcherTest.kt

```diff
@@ -91,4 +94,19 @@ class InstructionMatcherTest {
         val node = constNode add registerNode
         assertThat(instructionMatcher.findMatchesForValue(node, Register.VirtualRegister()).size).isEqualTo(1)
     }
+
+    @Test
+    fun `matcher sets function call label`() {
+        val instructionMatcher = InstructionMatcherImpl(emptyList(), listOf(CallPattern), emptyList())
+        val function = Definition.FunctionDeclaration(mockk(), "f", mockk(), listOf(), mockk(), mockk())
+        val node = CFGNode.Call(function)
+
+        val match = instructionMatcher.findMatchesForSideEffects(node).elementAt(0)
+        val instructions = match.instructionMaker(emptyMap())
+
+        assertThat(instructions.size == 1)
```

Comment:
This does not assert anything (change to assertTrue / add .isTrue()). The same happens two lines below.

Reply:
Or use the assertJ like it was designed and write `assertThat(instructions).hasSize(1)` 😉

Reply:
Yeah, I had a massive brain fart while writing this test